jib-core/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file.
 - New system property `jib.skipExistingImages` (false by default) to skip pushing images (manifests) if the image already exists in the registry. ([#2360](https://github.com/GoogleContainerTools/jib/issues/2360)
 
 ### Changed
+- Changed the default platform in jib-build-plan from `linux/amd64` to `amd64/linux`. ([#2597](https://github.com/GoogleContainerTools/jib/pull/2597)
```

Comment:
This is `jib-core`. You need to update the CHNANGELOG of `jib-build-plan`, because we are making changes to `jib-build-plan`. This CHANGELOG is for keeping the log of changes for jib-build-plan releases.

And how about wordings like this?

```
- Fixed the critical bug that the default `Platform` in `ContainerBuildPlan` has OS and architecture values switched with each other. ([#2597](https://github.com/GoogleContainerTools/jib/pull/2597)
```

And note that we add a blank line after `### Changed`. Don't forget the line when you change `jib-build-plan` CHANGELOG.

Reply:
I initially thought we have one CHANGELOG for the whole project and that's why I changed this one.Let me edit the correct one.